pytorch_toolkit/ote/ote/modules/exporters/mmdetection.py

```diff
@@ -40,6 +49,12 @@ def _export_to_openvino(self, args, tools_dir):
         should_run_alt_ssd_export = (hasattr(config.model, 'bbox_head')
                                      and config.model.bbox_head.type == 'SSDHead'
                                      and not config.get('nncf_config'))
+
+        if is_checkpoint_nncf and is_checkpoint_nncf(args['load_weights']):
+            # if the config does not contain NNCF part,
```

Comment:
how does this comment correlate to the code

Reply:
Fixed the comment -- made it more descriptive